src/dawn/engine_dawn.cc

```diff
@@ -37,6 +39,41 @@ namespace {
 // The minimum multiple row pitch observed on Dawn on Metal.  Increase this
 // as needed for other Dawn backends.
 const uint32_t kMinimumImageRowPitch = 256;
+const float kLodMin = 0.0;
+const float kLodMax = 1000.0;
+
+// Helper stuff
+struct Helper {
+  ::dawn::RenderPassColorAttachmentDescriptor*
+      cColorAttachmentsInfoPtr[kMaxColorAttachments];
```

Comment:
What is the "c" prefix on the name here, and "m" on mColorAttachmentsInfo?

We don't need those prefixes.